addons/mail/static/src/components/attachment_viewer/attachment_viewer.js

```diff
@@ -79,6 +80,7 @@ class AttachmentViewer extends Component {
 
     willUnmount() {
         document.removeEventListener('click', this._onClickGlobal);
+        this._imageRef.el && this._imageRef.el.removeEventListener('load', this._boundOnLoadImage);
```

Comment:
Don't use `&&` followed by instruction, it's a bit confusing. Just make a proper `if`.

Reply:
Also question: considering the listener is also added in `patched`, shouldn't it be also removed in `willPatch`?